server-apollo/src/sdk/withConflict.js

```diff
@@ -0,0 +1,57 @@
+const { SyncServerError, detectConflict } = require('./detectConflict')
+const { handleConflict } = require('./handleConflict')
+
+const dataSourceMappings = {
+  'knex': withConflictKnex
+}
+
+function withConflict(dataSource) {
```

Comment:
putting more thoughts into this maybe we can name this `updateWithConflict` it's missing the action